### PR TITLE
Add graphql-faker to wallet

### PR DESCRIPTION
**Pull request by Schmavery**
This checks in the full faked schema under schema.graphql and adds functionality (`npm run query-fake`) to generate the `graphql_schema.json` based on it. As a result, I removed the generated file and added it to the gitignore.
Also adds functionality to `npm run dev` to first start the fake graphql server.

Also updated the code to use an actual endpoint we support (checking node version).

### CODE REVIEW

File: frontend/wallet/package.json [hunk 1]
```diff
@@ -13,16 +13,20 @@
...
+    "fake": "graphql-faker --port 8080 -- schema.graphql",
+    "fake-inspector": "graphql-faker --open -- schema.graphql",
     "build": "bsb -make-world",
     "clean": "bsb -clean-world",
     "pack": "fpack --development ./lib/js/src/Index.js",
     "query": "send-introspection-query http://localhost:8080/graphql",
+    "query-fake": "concurrently 'npm run fake' 'sleep 5 && npm run query && echo You can now stop this process'",
     "reformat": "bsrefmt --in-place src/*.re",
-    "dev": "concurrently 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/Index.js' 'electron ./lib/js/src/App.js'",
+    "dev": "concurrently 'npm run fake' 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/Index.js' 'sleep 2 && electron ./lib/js/src/App.js'",
```
Comment (bkase) [inline frontend/wallet/package.json]: Do we need the `sleep 2`? Can we instead `(concurrently foo bar) && electron ./lib/js/src/App.js`?

Comment (Schmavery) [reply frontend/wallet/package.json]: I don't think so, because concurrently won't end on its own

Comment (bkase) [reply frontend/wallet/package.json]: it's 2 seconds not 2 millis right? Does this only happen once at startup or every change?

Comment (Schmavery) [reply frontend/wallet/package.json]: at startup, 2 seconds

**Pull request merged**